### PR TITLE
Updated empty survey response text to provide more detail and link to support article

### PR DESCRIPTION
**Pull request by hannahbergam**
This is to update the survey text to correctly convey the way survey results are aggregated and displayed. Currently, the text does not clarify that students must have answered the first question for it to be counted. 

Yes, the first image is from a different course, but you get the idea:)

Jira ticket: https://codedotorg.atlassian.net/browse/LP-1767

presurvey

image

### CODE REVIEW

File: apps/src/templates/sectionAssessments/SectionAssessments.jsx [hunk 1]
```diff
@@ -272,7 +273,9 @@ class SectionAssessments extends Component {
                   </div>
                 )}
                 {totalStudentSubmissions <= 0 && (
-                  <div>{i18n.emptySurveyOverviewTable()}</div>
+                  <div>
```
Comment (maureensturgeon) [inline apps/src/templates/sectionAssessments/SectionAssessments.jsx]: Is the wrapping `div` still necessary?

File: i18n/locales/source/blockly-mooc/common.json [hunk 2]
```diff
@@ -750,7 +750,7 @@
   "emptyFunctionBlocksErrorMsg": "The function block needs to have other blocks inside it to work.",
   "emptyFunctionalBlock": "You have a block with an unfilled input.",
   "emptySection": "Your section doesn't have any students! Go to the [Manage Students]({url}) tab to add some.",
-  "emptySurveyOverviewTable": "Because this survey is anonymous, we can only show aggregated results once there are at least 5 submissions.",
```
Comment (maureensturgeon) [inline i18n/locales/source/blockly-mooc/common.json]: I'm curious what's the difference between this common.json file and the en_us.json file? Is this one generated?

File: apps/i18n/common/en_us.json [hunk 3]
```diff
@@ -750,7 +750,7 @@
   "emptyFunctionBlocksErrorMsg": "The function block needs to have other blocks inside it to work.",
   "emptyFunctionalBlock": "You have a block with an unfilled input.",
   "emptySection": "Your section doesn't have any students! Go to the [Manage Students]({url}) tab to add some.",
-  "emptySurveyOverviewTable": "Because this survey is anonymous, we can only show aggregated results once there are at least 5 submissions.",
+  "emptySurveyOverviewTable": "Because this survey is anonymous, we can only show aggregated results once at least 5 students have submitted their survey and answered the first question of the survey. \n\nIf you have given this survey to your class and are still not seeing responses here, please give the survey to your class again and encourage your students to answer the first question of the survey before they submit the survey. [Learn more about surveys and assessments.](https://support.code.org/hc/en-us/articles/115001331951-Using-lock-settings-for-assessments-and-surveys)",
```
Comment (maureensturgeon) [inline apps/i18n/common/en_us.json]: This isn't in the ticket, but I'm wondering if we should remove the `en-us` from the link (change to: https://support.code.org/hc/articles/115001331951-Using-lock-settings-for-assessments-and-surveys), it looks like this link still works and the language gets automatically detected.

I was just thinking that if this is displayed in another language they wouldn't want to link to the support article in english.

Comment (hannahbergam) [reply apps/i18n/common/en_us.json]: ooo yes good point! Will do

File: i18n/locales/source/blockly-mooc/common.json (hunk 2 above)
Comment (hannahbergam) [reply i18n/locales/source/blockly-mooc/common.json]: Great question... I am unclear, to be honest. I believe putting it in en_us starts the translation pipeline, but just updated it both places to be safe. I can look into this!

File: apps/src/templates/sectionAssessments/SectionAssessments.jsx (hunk 1 above)
Comment (hannahbergam) [reply apps/src/templates/sectionAssessments/SectionAssessments.jsx]: Will fix now!

File: i18n/locales/source/blockly-mooc/common.json (hunk 2 above)
Comment (maureensturgeon) [reply i18n/locales/source/blockly-mooc/common.json]: Thanks! I think I've only been adding to `en_us.json`, so it'd be great to know if I need to add to both in the future

**Review by maureensturgeon — APPROVED** [review]
A couple of small changes, otherwise 👍

File: i18n/locales/source/blockly-mooc/common.json (hunk 2 above)
Comment (Hamms) [reply i18n/locales/source/blockly-mooc/common.json]: you're fine to only add it to `en_us.json`; the i18n build takes care of compiling it to everywhere it needs to go.

Comment (Hamms) [reply i18n/locales/source/blockly-mooc/common.json]: Note in particular that the history of the file has it being updated by an automated process https://github.com/code-dot-org/code-dot-org/commits/d42e1b8d01889e1f7b99ce8d682089e4f421c55f/i18n/locales/source/blockly-mooc/common.json

**Pull request merged**